sites/default/sqlconf.php

```diff
@@ -19,6 +22,13 @@
 $sqlconf["login"] = $login;
 $sqlconf["pass"] = $pass;
 $sqlconf["dbase"] = $dbase;
+$sqlconf["devmode"] = $devmode;
+$sqlconf["pooled"] = $pooled;
+$sqlconf["driver"] = $driver;
+
+// For use only by common/database/connector.php
+// TODO: Figure out why $sqlconf isn't in scope for common/database/connector.php
```

Comment:
@bradymiller any thoughts on this? I put this in for a workaround, but it is certainly not ideal :).